src/calculateRentalCost.js

```diff
@@ -29,6 +29,24 @@
 
 function calculateRentalCost(days) {
   // write code here
+
+  let cost = 0;
+
+  if (days >= 3 && days < 7) {
+    cost = days * 40 - 20;
+
+    return cost;
+  }
+
+  if (days >= 7) {
+    cost = (days * 40) - 50;
+
+    return cost;
+  } else {
+    cost = days * 40;
+
+    return cost;
+  }
```

Comment:
```suggestion
  if (days < 3) {
    return amount - smallDiscount;
  }

  if (days < 7) {

    return amount - bigDiscount;
  }

    return cost;
  }
```
Можно упростить таким образом запись